sapp/ui/server.py

```diff
@@ -99,5 +99,12 @@ def start_server(
     if static_resources:
         application.static_folder = static_resources
     if debug:
-        CORS(application, resources={r"/graphql": {"origins": "http://localhost:3000"}})
+        default_port = os.environ.get("PORT") or 3000
+        host = os.environ.get("HOST") or "localhost"
+        CORS(
+            application,
+            resources={
+                r"/graphql": dict(origins="http://{hostname}:{port}".format(hostname=host, port=default_port))
+            },
```

Comment:
Our linter triggered on this line - https://github.com/adamchainz/flake8-comprehensions#c408-unnecessary-dictlisttuple-call---rewrite-as-a-literal

```suggestion
            resources={
                r"/graphql": {"origins": "http://{hostname}:{port}".format(hostname=host, port=default_port)}
            },
```